api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Suite.java

```diff
@@ -52,8 +54,12 @@
   private static final Type SUITE_TYPE = new TypeToken<Suite>() {
   }.getType();
 
+  public void setCorrelationId(String correlationId) {
```

Comment:
why do we have the setter so high in the class?

Reply:
Fixed 😄